planterbox/feature.py

```diff
@@ -102,14 +102,36 @@ def load_examples(self, examples):
         if not examples:
             return
 
+        if FILE_PATH.match(examples[0]):
+            if examples[0][-4:] != '.csv':
+                raise Exception('Example file must be a csv file.')
+            else:
+                examples = self.read_file_into_examples(examples[0].strip())
+
         example_header = example_row(examples[0])
+
         for example in examples[1:]:
             example_data = example_row(example)
             yield {
                 label: datum for label, datum
                 in zip(example_header, example_data)
             }
 
+
+    def read_file_into_examples(self, fname):
+        import csv
+        import os
+        csv_examples = []
+
+        filename = os.path.dirname(__file__) + fname.strip()
+        with open(filename) as csv_file:
+            csv_reader = csv.reader(csv_file)
+            for row in csv_reader:
+                    csv_examples.append(row[0])
```

Comment:
If we're expecting the file to be |-separated example lines, we shouldn't be using `csv.reader()` here. Alternatively, if we're expecting the file to be a CSV, the tests should probably reflect that and this should probably reformat into the example data format...?